Item/Item.module.less

```diff
@@ -134,10 +143,14 @@
 			}
 		}
 
-		&.large { /* Needed to prevent global class being added in the DOM */ }
+		&.large {
+			height: @agate-item-large-height;
+			line-height: @agate-item-line-height-large;
```

Comment:
if you add line-height here, move the line-height used at line 49 here

Reply:
This property is not used anymore.